registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/dto/SyncResponseSuccessDto.java

```diff
@@ -6,15 +6,15 @@
 	private static final long serialVersionUID = 4422198670538094471L;
 
 	/** The registration id. */
-	private String registrationId;
+	private String packetId;

```

Comment:
this dto is also being used by old and new sync API. Please check carefully and create new dtos only for syncV2. I dont want any change in old sync api.


If you trace entire code the two methods "validateSync(for old sync)" and "validateSyncV2(for new sync)" calls same method - syncRegistrationRecord(). 
I would say you create one more dto - SyncResponseV2Dto which will have packetId as well(dont remove rid). Then in the old sync method (validateSync()) you convert from SyncResponseV2Dto to SyncResponseDto(without packet id.)

Reply:
registrationId is already present in SyncResponseDto from which both SyncResponseFailureDto and SyncResponseSuccessDto are extended.